### PR TITLE
SoXResampler

### PR DESCRIPTION
**Pull request by bramtayl**

### CODE REVIEW

File: S/SoX/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libsoxr", :libsoxr),
+    LibraryProduct("libsoxr-lsr", :libsoxr_lsr)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
```
Comment (giordano) [inline S/SoX/build_tarballs.jl:32]:
```
┌ Warning: To ensure that the correct version of libgomp is found at runtime, add the following entry to the list of dependencies of this builder
│ 
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/7H3La/src/auditor/compiler_abi.jl:4
```

```suggestion
dependencies = [
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
```

File: S/SoX/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SoX"
```
Comment (giordano) [inline S/SoX/build_tarballs.jl:5]: What about `SoXR` or an even more clear `SoXResampler`?  I googled "sox" and couldn't find _anything_ related to this package in the first 8 pages of results

Comment (bramtayl) [reply S/SoX/build_tarballs.jl]: Sure!

Comment (giordano) [reply S/SoX/build_tarballs.jl]: Note that the name of the directory should be the same (and I'm actually surprised it didn't fail)

Comment (bramtayl) [reply S/SoX/build_tarballs.jl]: Ok, thanks!

**Pull request merged**